packages/code-block/utils/prism-utils/index.test.js

```diff
@@ -9,7 +9,9 @@ const syntaxesList = Object.keys(syntaxes).reduce((acc, slug) => {
   return acc.concat(slug, aliases)
 }, [])
 
-describe('syntaxes', () => {
+// TODO: I'm not sure if we want to always run these tests, it seems like the list of syntaxes we provide should align with the needs of our
+// content, not necessarily all of the possible syntaxes exposed by the library.
+describe.skip('syntaxes', () => {
```

Comment:
@zchsh I'm offering this up for discussion. The upgrade to refractor introduced a bunch of new syntaxes, most of which seem pretty obscure. I'm thinking it might make more sense to update our mapping as our authors want to utilize new syntaxes, as opposed to doing it whenever we upgrade the underlying lib. Happy to continue with the current approach if we think it's a better way, though!

Reply:
Very on board with this - very much agreed we should focus on what syntaxes are actually in use. [Made a task for the platform board](https://app.asana.com/0/1100423001970639/1200801692661223/f) to assess what our new list of supported syntaxes would be 👍